tools/kochdocs.nim

```diff
@@ -112,15 +112,15 @@ proc getRst2html(): seq[string] =
   doAssert "doc/manual/var_t_return.rst".unixToNativePath in result # sanity check
 
 const
-  pdf = """
-doc/manual.rst
-doc/lib.rst
-doc/tut1.rst
-doc/tut2.rst
-doc/tut3.rst
-doc/nimc.rst
-doc/niminst.rst
-doc/gc.rst
+  rstList = """
```

Comment:
@a-mr `rstForPdf` would be better, more self documenting (it's only used for pdf, whereas `getRst2html` is used for html docs)

Reply:
=> https://github.com/nim-lang/Nim/pull/16991